packages/node_modules/@node-red/editor-client/src/js/ui/subflow.js

```diff
@@ -508,10 +508,8 @@ RED.subflow = (function() {
         var activeSubflow = RED.nodes.subflow(id);
 
         RED.nodes.eachNode(function(n) {
-            if (!keepInstanceNodes && n.type == "subflow:"+id) {
-                removedNodes.push(n);
-            }
-            if (n.z == id) {
+            if (n.z === id || !keepInstanceNodes && n.type === "subflow:" + id) {
+                RED.nodes.updateConfigNodeUsers(n, { emitEvent: false, action: "remove" });
```

Comment:
The event must (should) be triggered.
```suggestion
                RED.nodes.updateConfigNodeUsers(n, { action: "remove" });
```